g/golang-net/build_info.json

```diff
@@ -0,0 +1,17 @@
+{
+    "maintainer":"Vinod.K1@ibm.com",
+    "package_name" : "net", 
```

Comment:
@vinodk99 here package name should be "golang-net" .. This should match to directory path u use to add build script

Reply:
@nikhil-kalbande updated the build_info.json as requested.